storage/postgres/admin_storage_test.go

```diff
@@ -12,43 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package postgres_test
+package postgres
```

Comment:
Is it not possible to do black box testing of this? (just curious)

Reply:
Admin storage is actually currently blackbox tested; however I wanted to bring everything under the `postgres` package because 

A) that seems to be the pattern the rest of the codebase follows
B) It allows us to use a unified `TestMain` function.

Happy to change it back though, I don't have strong feelings about this